health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -78,34 +77,46 @@ export const UICustomizations = {
+          const microplanFileId = row?.campaignDetails?.additionalDetails?.microplanFileId || dummyFile;
           let options = [];
+
           if (row?.status == "DRAFT") {
             options = [{ code: "1", name: "Edit Setup" }];
           } else {
             options = [{ code: "1", name: "View Summary" }];
           }
+
+          const handleDownload = () => {
+            Digit.Utils.campaign.downloadExcelWithCustomName({ fileStoreId: microplanFileId, customName: t("Microplan Final Sheet") });
+          };
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)